<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Barcode_Tool_Ui</class>
 <widget class="QMainWindow" name="Barcode_Tool_Ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>197</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TurnFix - Barcode-Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/TurnFix.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="but_edit">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources/TurnFix.qrc">
        <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="but_sn">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources/TurnFix.qrc">
        <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="but_check">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources/TurnFix.qrc">
        <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="txt_change">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources/TurnFix.qrc"/>
 </resources>
 <connections/>
</ui>
